x-pack/metricbeat/magefile.go

```diff
@@ -154,7 +156,25 @@ func GoUnitTest(ctx context.Context) error {
 // Use RACE_DETECTOR=true to enable the race detector.
 func GoIntegTest(ctx context.Context) error {
 	return devtools.RunIntegTest("goIntegTest", func() error {
-		return devtools.GoTest(ctx, devtools.DefaultGoTestIntegrationArgs())
+		modulesFileInfo, err := ioutil.ReadDir("./module")
+		if err != nil {
+			return err
+		}
+
+		var failed bool
+		for _, fi := range modulesFileInfo {
+			if !fi.IsDir() {
+				continue
+			}
+			err := devtools.GoTest(ctx, devtools.GoTestIntegrationArgsForModule(fi.Name()))
```

Comment:
I'm wondering now, this will actually start the containers sequentially, but is it stopping them when the test ends?

Reply:
Nope, good catch. There wasn't such mechanism at all. I will add it here.

Reply:
After discussing it offline, we'll come back to this feature later.